test/ResponseTest.php

```diff
@@ -151,10 +151,18 @@ public function testConstructorIgonoresInvalidHeaders()
             'x-valid-string' => [ 'VALID' ],
             'x-valid-array' => [ 'VALID' ],
         ];
-        $response = new Response('php://memory', null, $headers);
+        $response = new Response('php://memory', 200, $headers);
         $this->assertEquals($expected, $response->getHeaders());
     }
 
+    /**
+     * @expectedException \InvalidArgumentException
```

Comment:
Please follow the same convention as the rest of tests. Use `setExpectedException` and fill the second argument.